tbp/drone/src/dji_tello/actions.py

```diff
@@ -0,0 +1,93 @@
+from numbers import Number
+
+from tbp.monty.frameworks.actions.actions import Action
+from tbp.monty.frameworks.actions.actuator import Actuator
+
+
+class TakeOff(Action):
+    @classmethod
+    def sample(cls, agent_id: str, sampler: "ActionSampler") -> "TakeOff":
```

Comment:
For future reference, you should be able to use `from __future__ import annotations` in order to be able to do things like self-referential type annotations without having to resort to strings. That combined with importing the `ActionSampler` class means you could rewrite this to be:

```suggestion
    def sample(cls, agent_id: str, sampler: ActionSampler) -> TakeOff:
```